network-api/networkapi/wagtailpages/forms.py

```diff
@@ -39,3 +39,15 @@ def clean(self):
             self.add_error('topics', 'Please select 2 topics max.')
 
         return cleaned_data
+
+
+# Similar validation to the BlogPageForm, but for the Blog Index page
+# and its field 'related_topics'.
+class BlogIndexTopicCountValidationForm(WagtailAdminPageForm):
```

Comment:
**Suggestion** Since you can only define one form per page type, I think `BlogIndexPageForm` would be the more appropriate form name.